packages/thumbprint-scss/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Removed
+
+-   [Major] Remove `function.scss` that contained the `tp-spacer` function.
```

Comment:
Could you add a note here on how to upgrade? Something along the lines of use `$tp-space-*` Tokens and hardcode values that don't fit in that system.

Reply:
Added.